policy-controller/src/main.rs

```diff
@@ -116,6 +129,8 @@ async fn main() -> Result<()> {
         default_detect_timeout: DETECT_TIMEOUT,
         probe_networks,
     });
+    let outbound_index = outbound_index::Index::shared(cluster_domain, default_opaque_ports);
```

Comment:
should the other index be called inbound_index?

In general, I'm not sure about the value in splitting the indexes like this, ultimately. We should probably look at alternatives.